src/Molecules/Pagination/Pagination.test.jsx

```diff
@@ -56,7 +56,7 @@ it('should call onSizeChange when user select another size', async () => {
 
   const { getByTestId, findByTestId } = render(
     <Wrapper>
-      <Pagination sizeOptions={[10, 20, 30]} onSizeChange={onSizeChange} />
+      <Pagination sizeOptions={[{ size: 10 }, { size: 20 }, { size: 30 }]} onSizeChange={onSizeChange} />

```

Comment:
That's another breaking change, is there any way to make it backward compatible ?

Reply:
I need the other PR to be merged in order to do that

Reply:
Done